src/main/java/racingcar/RacingCarApplication.java

```diff
@@ -0,0 +1,46 @@
+package racingcar;
+
+import racingcar.controller.RacingCarGame;
+import racingcar.domain.Cars;
+import racingcar.domain.Round;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.Scanner;
+
+/**
+ * RacingCarApplication.java
+ * 자동차 경주 게임
+ *
+ * @author Kimun Kim, github.com/tributetothemoon
+ * @author TaeHyeok Lee , github.com/taelee42
+ */
+public class RacingCarApplication {
+    public static void main(String[] arg) {
+        Scanner scanner = new Scanner(System.in);
+        InputView.setScanner(scanner);
+
+        Cars cars = createCarsFromUser();
+        Round round = createRoundFromUser();
+        RacingCarGame racingCarGame = RacingCarGame.createNewGame(cars, round);
+        racingCarGame.play();
+    }
+
+    private static Cars createCarsFromUser() {
+        try{
```

Comment:
```suggestion
        try {
```
자동 포맷팅을 사용하시거나 수시로 포맷팅을 하는 습관을 만들면 좋습니다 :)